ukrainian-phonetic/manifest.json

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2018 Artem Nagornyi. All rights reserved.
```

Comment:
Please change the line to the following:

Copyright 2018 The Extra Keyboards for Chrome OS Authors.

If you'd like you can add your name to the top-level AUTHORS file to reflect that you are a contributor to the project.

Reply:
Thank you. I updated the copyright.